bitnami/external-dns/Chart.yaml

```
@@ -28,4 +28,4 @@ maintainers:
 name: external-dns
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/external-dns
-version: 7.2.1
+version: 7.2.2
```

Please bump the minor version to add new values.
```suggestion
version: 7.3.0
```

---

Will do